clients/js/test/extensions/withdrawExcessLamport/withdrawExcessLamports.test.ts

```diff
@@ -0,0 +1,103 @@
+import { Account, generateKeyPairSigner } from '@solana/web3.js';
+import test from 'ava';
+import {
+  Token,
+  getWithdrawExcessLamportsInstruction,
+  fetchToken,
+} from '../../../src';
+import {
+  createDefaultSolanaClient,
+  createMint,
+  createToken,
+  generateKeyPairSignerWithSol,
+  sendAndConfirmInstructions,
+} from '../../_setup';
+
+test('it withdraws excess lamports from a token account', async (t) => {
+  // Given a destination account and a token account with excess lamports
+  const client = createDefaultSolanaClient();
+  const [authority, destination] = await Promise.all([
+    generateKeyPairSignerWithSol(client),
+    generateKeyPairSigner(),
+  ]);
+
+  // Create mint and token where authority is the owner
+  const mint = await createMint({
+    authority,
+    client,
+    payer: authority,
+  });
+
+  // Create token account owned by the authority
+  const token = await createToken({
```

Comment:
@lorisleiva I have used helper method for both `createMint()` and `createToken()`. Let me know if this is fine or if I need to change this to the longer version.